src/utils/types.ts

```diff
@@ -752,15 +757,15 @@ export type FilterTypeMethod = (filterKey: string, options?: FilterTypeOptions)
 
 export type FilterTypeDefinitions = {
   [filterType in
-    | 'number'
```

Comment:
Something is up with my formatter, sorry

Reply:
I re-auto-formatted that file for you: 0d703ed3

Reply:
thank you